diff --git a/dist/src/utils/eip-3770/config.js b/dist/src/utils/eip-3770/config.js
index 2a60ac262584a6cdae0dbdc3e73058be914c0f6c..0f3ffc6f3228af17be629233015a7779f92545e3 100644
--- a/dist/src/utils/eip-3770/config.js
+++ b/dist/src/utils/eip-3770/config.js
@@ -67,6 +67,8 @@ exports.networks = [
     { chainId: 288n, shortName: 'boba' },
     { chainId: 291n, shortName: 'orderly' },
     { chainId: 300n, shortName: 'zksync-sepolia' },
+    { chainId: 747n, shortName: 'flow-mainnet' },
+    { chainId: 545n, shortName: 'flow-testnet'},
     { chainId: 314n, shortName: 'filecoin' },
     { chainId: 321n, shortName: 'kcs' },
     { chainId: 322n, shortName: 'kcst' },
