diff --git a/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js b/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
index 00ae69d..6ef83de 100644
--- a/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
+++ b/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
@@ -29,6 +29,8 @@ exports.networks = [
     { chainId: 51n, shortName: 'txdc' },
     { chainId: 56n, shortName: 'bnb' },
     { chainId: 57n, shortName: 'sys' },
+    { chainId: 747n, shortName: 'flow-mainnet' },
+    { chainId: 545n, shortName: 'flow-testnet' },
     { chainId: 61n, shortName: 'etc' },
     { chainId: 63n, shortName: 'metc' },
     { chainId: 69n, shortName: 'okov' },
